### PR TITLE
Add first draft of analysis plan

### PR DESCRIPTION
**Pull request by matthew-brett**
Would be very grateful for review.

### CODE REVIEW

File: analysis_plan.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# First analysis plan
+
+This is the suggested analysis plan for the outlier detection.
```
Comment (pxr687) [inline analysis_plan.md:3]: Strong opening!

File: analysis_plan.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Reading we need to do
+
+* Look at Nipy's diagnostics (put in link here).
+* Review Principal Component Analysis (where?).
+* Review FSL's ICA method.
```
Comment (pxr687) [inline analysis_plan.md:9]:
```suggestion
* Review FSL's ICA method (need to discuss advantages/disadvantages vs PCA)
```

File: analysis_plan.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## Action plan
+
+* Use simplest possible outlier detection to get `find_outlier.py` to work.
+* Review methods above and try variants.
+* Review after course lessons on:
+    * Statistical detection of activation
+    * Motion detection
```
Comment (pxr687) [inline analysis_plan.md:17]:
```suggestion
    * Motion detection
    * Statistical detection of activation
    * Consider slice timing correction
```

File: analysis_plan.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# First analysis plan
+
+This is the suggested analysis plan for the outlier detection.
+
+## Reading we need to do
```
Comment (pxr687) [inline analysis_plan.md:5]: #5 we could maybe make this heading more formal?

**Review by pxr687 — COMMENTED** [review]
just a few suggestions/comments

File: analysis_plan.md (hunk 4 above)
Comment (matthew-brett) [reply analysis_plan.md:5]: Yes, good idea - let's put the discussion in the issue.

**Review by matthew-brett — COMMENTED** [review]
Some counter-comments.

**Pull request merged**